quantus/helpers/perturb_func.py

```diff
@@ -27,6 +27,32 @@
 )
 
 
+def perturb_batch(
+        perturb_func: Callable,
+        arr: np.ndarray,
+        indices: np.ndarray = None,
+        inplace: bool = False,
+        **kwargs,
+) -> Union[np.ndarray, None]:
+    """ perturbation of complete batch """
```

Comment:
Let's me consistent with the docstrings, following numpydoc https://numpydoc.readthedocs.io/en/latest/format.html